src/app.ts

```diff
@@ -78,28 +74,7 @@ main().catch(err => {
 });
 
 function populatePluginConfigFromEnv(config: PluginConfig, env: NodeJS.ProcessEnv) {
-  // Plugin v1 legacy env variables
-  if (env['GF_RENDERER_PLUGIN_TZ']) {
-    config.rendering.timezone = env['GF_RENDERER_PLUGIN_TZ'];
-  } else {
-    config.rendering.timezone = env['TZ'];
-  }
-
-  if (env['GF_RENDERER_PLUGIN_GRPC_PORT']) {
-    config.plugin.grpc.port = parseInt(env['GF_RENDERER_PLUGIN_GRPC_PORT'] as string, 10);
-  }
-
-  if (env['GF_RENDERER_PLUGIN_IGNORE_HTTPS_ERRORS']) {
-    config.rendering.ignoresHttpsErrors = env['GF_RENDERER_PLUGIN_IGNORE_HTTPS_ERRORS'] === 'true';
-  }
-
-  if (env['GF_RENDERER_PLUGIN_CHROME_BIN']) {
-    config.rendering.chromeBin = env['GF_RENDERER_PLUGIN_CHROME_BIN'];
-  }
-
-  if (env['GF_RENDERER_PLUGIN_VERBOSE_LOGGING']) {
-    config.rendering.verboseLogging = env['GF_RENDERER_PLUGIN_VERBOSE_LOGGING'] === 'true';
-  }
+  config.rendering.timezone = env['TZ'];
```

Comment:
Any reason keeping this one? Or maybe move it as fallback to https://github.com/grafana/grafana-image-renderer/blob/1d6e373363373ee4a4390979d031bc9243ce69e9/src/plugin/v2/grpc_plugin.ts#L204 ?